mod/workspace/cache.js

```diff
@@ -1,36 +1,36 @@
-const file = require('../provider/file')
+const file = require("../provider/file");
 
-const cloudfront = require('../provider/cloudfront');
+const cloudfront = require("../provider/cloudfront");
 
-const http = require('./httpsAgent')
+const http = require("./httpsAgent");
 
 const getFrom = {
-  'https': ref => http(ref),
-  'file': ref => file(ref.split(':')[1]),
-  'cloudfront': ref => cloudfront(ref.split(':')[1]),
-}
+  https: (ref) => http(ref),
+  file: (ref) => file(ref.split(":")[1]),
+  cloudfront: (ref) => cloudfront(ref.split(":")[1]),
+};
 
-const assignTemplates = require('./assignTemplates')
+const assignTemplates = require("./assignTemplates");
 
-const defaults = require('./defaults');
+const defaults = require("./defaults");
 
-const assignDefaults = require('./assignDefaults');
+const assignDefaults = require("./assignDefaults");
 
 let workspace = null;
```

Comment:
Do we really need to declare `workspace` outside the returned function?

Reply:
I'm not sure why this is this way, maybe bc it uses on the other two functions too?